tendrl/common/utils/command.py

```diff
@@ -5,9 +5,25 @@
 ANSIBLE_MODULE_PATH = "core/commands/command.py"
 LOG = logging.getLogger(__name__)
 
+SAFE_COMMAND_LIST = [
```

Comment:
Should we externalize these command in tendrl owned yaml file? Just a thought, but if we are very strict on list of commands supported, then it ok to keep inside code. My only point is just for a new command we would need a code change later.

Comment??